src/p5.dance.js

```diff
@@ -875,6 +875,13 @@ module.exports = class DanceParty {
     this.setPropEach(group, "dance_speed", val);
   }
 
+  changePropEachBy(group, property, val) {
```

Comment:
Instead of adding a new function to handle the group case, can we just make the `changePropBy` function able to handle either individual sprites or groups? And same for all the other functions that currently have two versions.

Reply:
Talked offline. Going to track and follow-up in separate PR: https://codedotorg.atlassian.net/browse/STAR-770